From f06cce42e4b881261fecf85ca4e4c6b9d52397ab Mon Sep 17 00:00:00 2001
From: Martin Jansa <Martin.Jansa@gmail.com>
Date: Sat, 12 Aug 2023 22:52:12 +0200
Subject: [PATCH] runner-debug: use WEBOS_INSTALL_BINS as other binaries

* $$WEBOS_INSTALL_WEBOS_SDKDIR/bin depends on webos_sdkdir to be defined
  to some non-empty value:
  recipe-sysroot/usr/lib/mkspecs/features/webos-variables.prf:webosSet(WEBOS_INSTALL_WEBOS_SDKDIR, $$(webos_sdkdir))

  but it's not set anywhere in meta-webos-ports and this ends just
  "/bin" causing QA installed-vs-shipped when usrmerge is used:

ERROR: qml-webos-framework-1.0.0-161+gitAUTOINC+a6969a4cb6-r35 do_package: QA Issue: qml-webos-framework: Files/directories were installed but not shipped in any package:
  /bin/qml-runner-dev
Please set FILES such that these items are packaged. Alternatively if they are unneeded, avoid installing them or delete them within do_install.
qml-webos-framework: 1 installed and not shipped files. [installed-vs-shipped]

Signed-off-by: Martin Jansa <Martin.Jansa@gmail.com>
Upstream-Status: Pending

---
 tools/runner-debug/runner-debug.pro | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/runner-debug/runner-debug.pro b/tools/runner-debug/runner-debug.pro
index 49c4f01..a05fd30 100644
--- a/tools/runner-debug/runner-debug.pro
+++ b/tools/runner-debug/runner-debug.pro
@@ -21,6 +21,6 @@ include(../runner/runner.pri)
 
 CONFIG += qml_debug
 
-target.path = $$WEBOS_INSTALL_WEBOS_SDKDIR/bin
+target.path = $$WEBOS_INSTALL_BINS
 
 INSTALLS += target
